packages/ChangesReloaded-Core.package/CRBrowser.class/class/open.st

```diff
@@ -1,4 +1,4 @@
 instance-creation
 open
 
-	ToolBuilder open: self new
\ No newline at end of file
+	^ ToolBuilder open: self new
```

Comment:
Looks like the linter (?) expects a new line?
Appears multiple times in this Branch